test/queries/validator_search_query_test.rb

```diff
@@ -1,81 +1,95 @@
 # frozen_string_literal: true
 
-require 'test_helper'
+require "test_helper"
 
 class ValidatorSearchQueryTest < ActiveSupport::TestCase
   def setup
     super
-
-    @validators = [
-      create(:validator, :with_score, name: 'Name1'),
-      create(:validator, :with_score, name: 'Name2'),
-      create(:validator, :with_score, account: 'Name1Account'),
-      create(:validator, :with_score, account: 'Name2Account'),
-      create(:validator, :with_score, name: 'Name1', network: 'searchnet'),
-      create(:validator, :with_score, name: 'Name2', network: 'searchnet'),
-      create(:validator, :with_score, account: 'Name1Account', network: 'searchnet'),
-      create(:validator, :with_score, account: 'Name2Account', network: 'searchnet'),
-      create(:validator)
+    # Setup data centers with data center hosts
+    data_center_name1 = create(:data_center, :china)
+    data_center_name1.update_column(:data_center_key, "Name1") # Update to match query
```

Comment:
Can it be done in create?

Reply:
data_center_key is created in the before_save callback, so it's not what I needed